### PR TITLE
feat: Followers feature

### PR DESCRIPTION
**Pull request by DurmazDev**

### CODE REVIEW

**Comment by gecicidegisken** [comment]
We prefer PR messages to follow conventional commits as well as commit messages. Also commit messages need to be in [imperative mood](https://git.kernel.org/pub/scm/git/git.git/tree/Documentation/SubmittingPatches?h=v2.36.1#n181). Maybe you can consider changing commit messages in this pull request and/or stick to the convention for following commits

File: backend/app/resources/feed.py [hunk 1]
```diff
@@ -27,6 +28,7 @@ def get(self):
         sort_vote = request.args.get("vote", None, type=str)
         sort_date = request.args.get("date", "desc", type=str)
         sort_tag = request.args.get("tag", None, type=str)
+        followed_only = request.args.get("q", None, type=str)
```
Comment (gecicidegisken) [inline backend/app/resources/feed.py:31]: General comment but I will just mention it here, it's best to always review the docstrings after changing a code snippet. We always need our docstrings _(which are the primary sources to interact with the api in this context)_ to be up-to-date

File: backend/app/resources/post.py [hunk 2]
```diff
@@ -325,3 +325,42 @@ def delete(self, id):
                 return {"message": "Tag removed successfully."}, 204
             else:
                 return {"error": "No tag found with this id in post tags."}, 404
+
+def getOtherUsersPostsWithUserID(id):
```
Comment (gecicidegisken) [inline backend/app/resources/post.py:329]: PEP8 requires function or method names to be in _snake_case_
[Refer here](https://peps.python.org/pep-0008/#function-and-variable-names)

**Comment by DurmazDev** [comment]
I tried the process according to the document you sent and it seems to work. I think I still have a problem with the "Imperative mood" part, I will be more careful in the next commits.

**Pull request merged**